### PR TITLE
🔀 Add seed images & Update assets API

### PR DESCRIPTION
**Pull request by inDream**
Connects #62

### CODE REVIEW

File: server/seeders/fixture.json [hunk 1]
```diff
@@ -1,5 +1,31 @@
 {
   "licenses": ["cc0", "cc-by", "cc-by-sa", "cc-by-nd"],
+  "licensesUrl": [
+    "https://creativecommons.org/publicdomain/zero/1.0/",
+    "https://creativecommons.org/licenses/by/4.0/",
+    "https://creativecommons.org/licenses/by-sa/4.0/",
+    "https://creativecommons.org/licenses/by-nd/4.0/"
+  ],
```
Comment (nwingt) [inline server/seeders/fixture.json]: Suggest merging `licensesUrl` into `licenses`
```javascript
[
  {
     name: '',
     url: '',
  },
  ...
]
```

File: server/seeders/20180615101955-demo-asset.js [hunk 2]
```diff
@@ -1,21 +1,66 @@
...
+          const wallet = `address ${(index % 3) + 1}`;
+
+          const mediaObj = {
+            creator: [wallet],
+            dateCreated: indexedDate,
+            description: '',
+            fingerprint: hash256,
+            footprint: [],
+            ipfs: ipfsAdd[0].hash,
+            license: licensesUrl[index % licensesUrl.length],
```
Comment (nwingt) [inline server/seeders/20180615101955-demo-asset.js]: Please explain why assign `licensesUrl` here

Comment (inDream) [reply server/seeders/20180615101955-demo-asset.js]: https://github.com/likecoin/puttyimages-web/blob/master/server/api/assets.js#L112

File: server/api/assets.js [hunk 3]
```diff
@@ -126,11 +130,13 @@ router.post(
...
             {
               fingerprint: hash256Bytes,
+              height,
               ipfs: bs58.decode(ipfsAdd[0].hash),
               ipld: bs58.decode(ipld.toBaseEncodedString()),
               license,
               tags: [{ name: 'hehe' }, { name: 'haha' }],
               wallet,
+              width,
             },
```
Comment (mileswong) [inline server/api/assets.js]: Missing `description`?

Comment (mileswong) [reply server/api/assets.js]: Are tags ready for assets upload? Since I see it saves `hehe` and `haha` instead of the real tags submitted by user

Comment (inDream) [reply server/api/assets.js]: Updated thanks, I have not tested it as there's no UI now.

**Pull request merged**